### DISCUSSION TITLE
Can I use layouts partials other than "sidebar", "footer", "header"?

### DISCUSSION THREAD

**Discussion by inkscar** [datenstrom/community#410] [Ask a question] [answered]
I guess one can produce a modular-style page by listing the content of subpages, but I tried and failed to use layout in an analogous way as that for the sidebar and footer to do that. Can one have other (differently named) layouts with related content file to display the same way?

**Comment by markseuffert** [comment, marked as answer]
I think the API has everything you need. First access a content file `example.md` in the current folder with `$page = $this->yellow->content->shared("example")`. Then output the HTML code in your layout with `echo $page->getContent()`.  There's no limit for the number of subpages or layouts. 

I would start with the API, it comes with many examples. It should take only a few days until a developer can make own layouts. You can post your code here on GitHub, if you have specific technical questions. You can also contact the commercial support, if you want to hire someone to make a layout.

Hope this helps and happy coding. :smile:

**Comment by inkscar** [comment]
Thanks! I'll install the dev version later and have a look.
